components/src/Modal/Modal.js

```diff
@@ -11,10 +11,11 @@ import { PreventBodyElementScrolling } from "../utils";
 
 const ModalContent = styled.div({
   marginTop: "-64px",
-  marginBottom: "-80px",
+
```

Comment:
can you remove the space here? 
Also I'm not quite clear on why these margins and paddings are changing, are they just style tweaks or specific changes for the select?

Reply:
The new values allow more space for the select options. Also, I corrected some incorrect spacing where ModalContent was 8 or 16px larger than its parent - StyledReactModal.